### DISCUSSION TITLE
Two questions about the 2FA that will be implemented soon

### DISCUSSION THREAD

**Discussion by Rod-in-NM** [community/community#50653] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

(I hope this is the appropriate place to ask this question. Please forgive me if I've posted this question in the wrong place.)

Today I got an email from GitHub, informing me that I need to enable 2FA for my GitHub account within 45 days. I'm OK with that, but I've got two questions which I need answered, before I proceed with this.

## Can I use the GitHub Mobile app with two GitHub accounts?
I have two personal GitHub accounts; one is Pro and the other is a free account. I created both for different purposes (the Pro account for my personal use and the free account loosely associated with my work). According to the email and blog post, one method I can use for 2FA is the GitHub Mobile app. I have that app on my phone, but at the moment it is only associated with my GitHub Pro account. Is it possible to use the GitHub Mobile app, with more than one GitHub account?

## Can I use one phone number for SMS messages for 2FA?
This question is like the previous one. Again, I only have one phone, so I use that for both accounts (I think; I'll have to double check to see if I've even entered any phone number with either account). Anyway, may I use the same phone number for different GitHub accounts?

**Comment by asb-realrev** [comment, marked as answer]
Same question here: 
https://github.com/orgs/community/discussions/74606 

As replied there too: 
I have an ignored (3+ weeks) open support cases with Github on mandatory 2FA (with threatened lockouts) since I was able to configure SMS 2FA successfully with one account, but the other second one keeps on failing during Github's enrollment without any details other than it "failed".
So I'm wondering if that limitation perhaps might be Github's internal bug, with zero helpful failure/error messages nor any documentation of those additional requirements.

If others already know or are aware of this limitation, PLEASE share!

**Reply by asb-realrev** [reply]
https://github.com/orgs/community/discussions/44020 
Came across this post which suggests that at some point SMS for multiple accounts to the same phone number DID work and was allowed (start of 2023).

**Comment by gil-son** [comment]
Yes,

You can use the Authenticator app and create two validations to each account

**Reply by Rod-in-NM** [reply]
Thank you for this information, @gil-son
